plebnet/agent/core.py

```diff
@@ -36,6 +36,33 @@
     'simple_moving_average': SimpleMovingAverage
 }
 qtable = None  # Used to store the QTable of the agent and only load once.
+
+remote_tables = []  # list to store the remote qtables
+
+
+# TODO: initiate consumer in setup or somewhere it could be "always listening"
+class LearningConsumer(messaging.MessageConsumer):
+
+    def parse_message(self, raw_message: messaging.Message):
+        """
+         Parses a raw message into command and data.
+         raw_message: raw_message to parse
+        """
+        channel = raw_message["channel"]
+        command = raw_message['command']
+        data = raw_message['data']
```

Comment:
Why not direct access to properties? Message is an object, not a dictionary

Reply:
you are right, let me change it